test/parallel/test-repl-preview-result.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+const common = require('../common');
+const ArrayStream = require('../common/arraystream');
+const assert = require('assert');
+const repl = require('repl');
+const inputStream = new ArrayStream();
+const outputStream = new ArrayStream();
+const input = [
+  '"  test ".trim()',
+  'var a=[9,0]; a.sort()',
+  '[9,0].sort()',
+  '6 + 10'
+];
+let output = '';
+
+outputStream.write = (data) => { output += data.replace('\r', ''); };
+
+const r = repl.start({
+  prompt: '',
+  input: inputStream,
+  output: outputStream,
+  terminal: true,
+  useColors: false
+});
+
+r.on('exit', common.mustCall(() => {
+  const actual = output.split('\n');
+
+  const wrapWithColorCode = (str) => `\u001b[90m //${str}\u001b[39m`;
+
+  assert.ok(actual[0].includes(wrapWithColorCode('\'test\'')));
+  // side effect scenario, preview is not available
+  assert.ok(!actual[2].includes('// '));
```

Comment:
`appendPreview` doesn't have a space character after `//`. Is this intentional?

Reply:
Nope, it was a miss. Good catch. Will fix them.